### PR TITLE
Update gradle / minSDK parsing for Android builds

### PR DESCRIPTION
**Pull request by AndyScherzinger**
Update gradle to 5.4.1
Update minSDK replacement to 16

Signed-off-by: Andy Scherzinger <info@andy-scherzinger.de>

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
cc @tobiasKaminsky for review

File: android/gradle/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,7 +1,6 @@
-#Thu Apr 18 07:07:10 CEST 2019
+#Wed Aug 21 07:03:37 CEST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
-distributionSha256Sum=53b71812f18cdb2777e9f1b2a0f2038683907c90bdc406bc64d8b400e1fb2c3b
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```
Comment (tobiasKaminsky) [inline android/gradle/gradle/wrapper/gradle-wrapper.properties]: Don't we need a checksum here?

Comment (AndyScherzinger) [reply android/gradle/gradle/wrapper/gradle-wrapper.properties]: Checksums have been removed by Google recently (!)
see your commit https://github.com/nextcloud/android/commit/4411f22cd5b2f060dfc7e5130abf8364eeae56f1

Comment (tobiasKaminsky) [reply android/gradle/gradle/wrapper/gradle-wrapper.properties]: True :see_no_evil:

**Comment by tobiasKaminsky** [comment]
Fixed build.gradle so that docker works again.
(was shown here as status, so let us wait for confirmation)

**Pull request merged**